nb/import_pipeline.ipynb

```diff
@@ -0,0 +1,182 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Importing macauff results into HiPSCat / LSDB\n",
+    "\n",
+    "The macauff pipeline OUTSIDE of LSDB will create a series of yml and csv files\n",
+    "that represent the counterpart assignments and likelihoods between objects in \n",
+    "two catalogs.\n",
+    "\n",
+    "To convert into an LSDB-friendly association table, you will need:\n",
+    "- match file schema file (either yaml or parquet schema)\n",
+    "- match output CSVs\n",
+    "- pre-hipscatted left catalog\n",
+    "- pre-hipscatted right catalog"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Schema\n",
+    "\n",
+    "If you have a yaml schema, you will first need to convert it into a parquet\n",
+    "schema to be understood by the import pipeline.\n",
+    "\n",
+    "Use the `from_yaml` utility method to read the yaml files; parse column names,\n",
+    "types, and other key-values; convert into parquet types; write to an empty\n",
+    "parquet file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from hipscat.io import file_io\n",
+    "from hipscat_import.cross_match.macauff_metadata import from_yaml\n",
```

Comment:
This moved to `lsdb_macauff` now I think, so this is `from lsdb_macauff.import_pipeline.convert_metadata import from_yaml` or thereabouts, presumably